lib/galaxy/datatypes/sequence.py

```diff
@@ -20,7 +20,7 @@
 from galaxy.datatypes.metadata import MetadataElement
 from galaxy.datatypes.sniff import get_headers
 from galaxy.util import nice_size
-from galaxy.util.checkers import is_gzip
+from galaxy.util.checkers import is_gzip, is_bz2
```

Comment:
Thanks for the explanation @nsoranzo.
I had been wondering why the second option is preferred, but that makes a lot of sense.